x/distribution/types/genesis.go

```diff
@@ -67,3 +71,26 @@ func DefaultGenesisWithValidators(valAddrs []sdk.ValAddress) GenesisState {
 		DelegationDistInfos: ddis,
 	}
 }
+
+// ValidateGenesis validates the genesis state of distribution genesis input
+func ValidateGenesis(data GenesisState) error {
+	if data.CommunityTax.IsNegative() || data.CommunityTax.GT(sdk.OneDec()) {
+		return fmt.Errorf("mint parameter CommunityTax should be positive "+
+			"and less than one, is %s", data.CommunityTax.String())
```

Comment:
This tests less than or equal to one, not strictly less than - which did you intend?

Reply:
updated to `mint parameter CommunityTax should non-negative and less than one`